samcli/commands/test_runner/init/cli.py

```diff
@@ -233,3 +250,56 @@ def query_tagging_api(tags: dict, boto_client_provider: BotoProviderType) -> Uni
     )
 
     return [resource.get("ResourceARN") for resource in resource_tag_mapping_list]
+
+
+def _extract_action(iam_action_string: str) -> str:
+    indx = iam_action_string.index(":")
```

Comment:
Is there a reason for `indx` as opposed to `index`?

Reply:
nope, will fix :)